pkg/bbgo/order_executor_general.go

```diff
@@ -162,6 +163,27 @@ func (e *GeneralOrderExecutor) BindProfitStats(profitStats *types.ProfitStats) {
 	})
 }
 
+func (e *GeneralOrderExecutor) BindProfitTracker(profitTracker *report.ProfitTracker) {
```

Comment:
hmm move this to the strategy? I think it's bit too much for the order executor

Reply:
done